package.json

```diff
@@ -31,7 +31,8 @@
         "npm-pack": "node common/scripts/install-run-rush.js npm-pack --verbose",
         "npm-publish": "node ./tools/release-tools/npm_publish.js",
         "npm-set-latest": "node ./tools/release-tools/npm_set_latest.js",
-        "gh-status": "node ./tools/status-tools/github-status.js"
+        "gh-status": "node ./tools/status-tools/github-status.js",
+        "script-inject": "node tools/github-page-script-injection/injectScript.js"
```

Comment:
I do not have access to change github action currently, later could add  steps:
      - name: Checkout
        uses: actions/checkout@v4
      - name: Inject Script
        run:  node ./injectScript.js in github action for page deployment so that the inject could happen right before the doc website is published.

Reply:
I don't think there is any current github action to do this. It (should) just be a case that once we "generate" the docs we then just run this new task.

Reply:
I am thinking about add it to github action because the run takes after the file is uploaded and before the web is published, the page file is not changed. which means subsequent runs are not influenced by the changes made in previous runs. Presently, we execute 'npm run api-docs' in each folder for documentation updates. If we transition to running this command at the root level, taking into account the script injection at the root folder, it might simplify tracking changes without impacting subsequent runs.

Reply:
Well actually, we just run "rush api-docs" an it runs the api-docs in each project. We could just as easily ass a "root" npm script in the root to 
`"api-docs": "rush api-docs && npm run script-inject"`

Then we just run `npm run api-docs` from the root and everything is done. I also don't believe that we can "update" the uploaded "docs" via a github action because the `main` branch is protected so it always needs a PR to push changes.